### DISCUSSION TITLE
Visualization

### DISCUSSION THREAD

**Discussion by techcatgato** [pybricks/support#1779] [Q&A]
Is there any way to make a display while spike prime is running? Like display some objects and a board on the screen?

**Comment by laurensvalk** [comment]
You can display basic things on the 5x5 display on the SPIKE hub, but we do not have a visual canvas in the app if that's what you mean.

**Reply by techcatgato** [reply]
Any way to to use visual canvas by using vsc instead of code.pybricks?

**Reply by laurensvalk** [reply]
Not something ready to use, no. We're working on things similar to that, but it won't be ready any time soon.
